### PR TITLE
minor updates to get teacher UI rendering

### PR DESCRIPTION
**Pull request by nathanstitt**

### CODE REVIEW

File: tutor/src/routes.js [hunk 1]
```diff
@@ -47,11 +47,9 @@ const getRoutes = (router) => {
                             ],
                         },
                     ],
-                }, {
-                    path: 'metrics/:id', name: 'reviewTask',
-                    renderer: r(() => import('./screens/teacher-review-metrics')) },
```
Comment (jivey) [inline tutor/src/routes.js:52]: Oh yeah we removed this screen right?

File: tutor/src/screens/assignment-review/ux.js [hunk 2]
```diff
@@ -312,9 +315,11 @@ export default class AssignmentReviewUX {
 
     @action.bound async onSavePlan() {
         await this.editUX.savePlan();
-        Object.assign(this.planScores, pick(this.editUX.plan, ['title', 'description']));
-        this.planScores.grading_template = this.editUX.plan.gradingTemplate;
-        this.isDisplayingEditAssignment = false;
+        runInAction(() => {
```
Comment (jivey) [inline tutor/src/screens/assignment-review/ux.js:318]: I didn't realize we'd need to use `runInAction` inside something decorated with `@action`? Or is this to process everything a single transaction?

**Review by jivey — APPROVED** [review]

File: tutor/src/screens/assignment-review/ux.js (hunk 2 above)
Comment (nathanstitt) [reply tutor/src/screens/assignment-review/ux.js:318]: yah this is a tricky one, the function is async, which means babel will actually splits it into multiple functions, and the code after the `await` will not be enclosed inside the main action.   https://mobx.js.org/actions.html#asynchronous-actions explains it a bit more

Comment (nathanstitt) [reply tutor/src/screens/assignment-review/ux.js:318]: there's a new way to do this using `flow` but I haven't explored it yet

**Pull request merged**